src/main/java/org/sonarlint/intellij/config/project/ProjectExclusionsPanel.java

```diff
@@ -56,7 +56,7 @@ public ProjectExclusionsPanel(Project project) {
         + "\">override</a> your locally defined exclusions.";
       editorPane2.setVisible(false);
     } else {
-      message = "When a project is connected to SonarQube or SonarCloud, exclusions defined in the server's General Settings " +
+      message = "When a project is connected to SonarQube (Server, Cloud), exclusions defined in the server's General Settings " +
```

Comment:
Not sure if `server's General Settings` is still valid because SQ is now called SQ-S. Maybe have `... exclusions defined in its General Settings ...`?